src/Microsoft.AspNetCore.Routing/DispatcherMiddleware.cs

```diff
@@ -64,7 +64,11 @@ internal sealed class DispatcherMiddleware
             await matcher.MatchAsync(httpContext, feature);
             if (feature.Endpoint != null)
             {
-                Log.EndpointMatched(_logger, feature);
+                Log.MatchSuccess(_logger, feature);
+            }
+            else
+            {
+                Log.MatchFailure(_logger);
```

Comment:
RoutingMiddleware only logged on failure. DispatcherMiddleware (previously) only logged on success. What is correct?

Reply:
I agree on Debug. Can review logging later.